README.md

```diff
@@ -1139,6 +1139,8 @@ Below are listed all the variables you can customize (you may also want to take
       # Django middleware setting to add "CorsPostCsrfMiddleware".
       # By default, it is set to false.
       replace_https_referer: true
+    # The default retention policy that applies to the timeseries data
+    openwisp2_monitoring_default_retention_policy: "26280h0m0s" # 3 years
```

Comment:
I would add this before the NGINX related settings in the README.  It will be great if we can add the link to the README of openwisp-monitoring for this setting.

Reply:
Sure, I'll update that :+1: